src/natcap/invest/crop_production_regression.py

```diff
@@ -24,75 +26,138 @@
-                    "potassium_rate"
-                ],
-            },
             "type": "csv",
-            "required": True,
+            "columns": {
+                "crop_name": {"type": "freestyle_string"},
+                "nitrogen_rate": {
+                    "type": "number",
+                    "units": u.kilogram/u.hectare
+                },
+                "phosphorus_rate": {
+                    "type": "number",
+                    "units": u.kilogram/u.hectare
+                },
+                "potassium_rate": {
+                    "type": "number",
+                    "units": u.kilogram/u.hectare
+                }
+            },
             "about": (
-                "A table that maps fertilization rates to crops in the "
-                "simulation.  Must include the headers 'crop_name', "
-                "'nitrogen_rate',  'phosphorous_rate', and "
-                "'potassium_rate'."),
+                "A table that maps crops to fertilization rates for nitrogen, "
+                "phosphorus, and potassium."),
             "name": "Fertilization Rate Table Path"
         },
         "aggregate_polygon_path": {
-            "type": "vector",
-            "required": False,
-            "about": (
-                "A polygon vector containing features with which to "
-                "aggregate/summarize final results. It is fine to have "
-                "overlapping polygons."),
-            "name": "Aggregate results polygon"
+            **utils.AOI_ARG,
+            "required": False
         },
         "model_data_path": {
-            "validation_options": {
-                "exists": True,
-            },
             "type": "directory",
-            "required": True,
+            "contents": {
+                "climate_regression_yield_tables": {
+                    "type": "directory",
+                    "contents": {
+                        "(\\w+)_regression_yield_table.csv": {
+                            "items": "One for each crop",
+                            "type": "csv",
+                            "columns": {
+                                'climate_bin': {"type": "code"},
+                                'yield_ceiling': {
+                                    "type": "number",
+                                    "units": u.metric_ton/u.hectare
+                                },
+                                'b_nut':  {"type": "number", "units": u.none},
+                                'b_k2o':  {"type": "number", "units": u.none},
+                                'c_n':    {"type": "number", "units": u.none},
+                                'c_p2o5': {"type": "number", "units": u.none},
+                                'c_k2o':  {"type": "number", "units": u.none}
+                            }
+                        }
+                    }
+                },
+                "crop_nutrient.csv": {
```

Comment:
They could share, but it would probably have to go in utils? I think it's ok to stay duplicated here, especially since it's a useful reference while you're looking at the model.